### ISSUE TITLE
[BUG] Delivery options don't show up when MSI quantity is 1

### ISSUE THREAD

**Issue by edwinbos** [tig-nl/postnl-magento2#223]
**To Reproduce**
1. Add a product with a MSI quantity of 1
2. Go to the checkout and choose PostNL as a shipping method

**Expected result**
Delivery opties should be shown

**Actual result**
Delivery opties don't show up

**Workaround**
In _tig/postnl-magento2/Service/Quote/CheckIfQuoteItemsAreInStock.php (L:241)_ in the function _getStockFromSources()_ the code is:

```
if (($sourceQty - $minimumQuantity) > $requiredQuantity) {
    return true;
}
```

should be:
```
if (($sourceQty - $minimumQuantity) >= $requiredQuantity) {
    return true;
}
```

Because if there is just 1 item the check is (1 - 0 > 1), which is false.

**Please complete the following information**
- Magento version: 2.3.5
- PHP version: 7.3
- TIG PostNL version: 1.9.5

**Comment by tig-jaspersmits** [comment]
Hi @edwinbos ,

Thanks for sharing this with us. I'll test this and inform our team about this.
Will let you know once we've added this to the backlog.

Kind regards,
Jasper

**Comment by tig-jaspersmits** [comment]
Hi @edwinbos ,

Just added this to the backlog.
Keep an eye on the release notes here so you can see when we release a new version including this fix.

Kind regards,
Jasper

**Comment by tig-vincentthart** [comment]
Hi @edwinbos,

This issue has been resolved in version 1.9.6.

Kind regards,
Vincent
